paasta_tools/cli/cmds/logs.py

```diff
@@ -167,7 +168,24 @@ def prefix(input_string, component):
     return "%s: %s" % (LOG_COMPONENTS[component]['color'](component), input_string)
 
 
-def paasta_log_line_passes_filter(line, levels, service, components, clusters):
+def check_timestamp_in_range(timestamp, start_time, end_time):
+    """A convenience function to check if a timestamp is within the given start and end times,
+    returns true if start_time or end_time is None
```

Comment:
That makes sense, and it'd be a pretty easy change. Plus there's convenient `datetime.datetime.min/max` constants